code/create_enigma_terms_json.py

```diff
@@ -73,22 +81,58 @@ def create_terms_json(
 
 
 def main():
-    vocab_gsheet = fetch_gsheet(VOCAB_GSHEET_ID)
-    vocab_df = pd.DataFrame(vocab_gsheet.get_all_records(default_blank=None))
+    parser = argparse.ArgumentParser(
+        description="Generate standardized terms vocabulary JSON files from Google Sheets tables for a Neurobagel community configuration."
+    )
+    parser.add_argument(
+        "community_config_dir",
+        type=Path,
+        help="Path to a community configuration directory.",
+    )
+    args = parser.parse_args()
+
+    if not args.community_config_dir.is_dir():
+        print(
+            f"Community directory does not exist: {args.community_config_dir}"
+        )
+        sys.exit(1)
+    if not (
+        args.community_config_dir / COMMUNITY_TERMS_MANIFEST_FILE
+    ).is_file():
+        print(
+            f"{COMMUNITY_TERMS_MANIFEST_FILE} not found in {args.community_config_dir}"
+        )
+        sys.exit(1)
```

Comment:
:cherries: would be a bit cleaner if you factored out the definition of the parser to its own function